commonmark/src/main/java/org/commonmark/internal/ReferenceParser.java

```diff
@@ -1,11 +0,0 @@
-package org.commonmark.internal;
-
-/**
- * Parser for inline references
- */
-public interface ReferenceParser {
```

Comment:
FYI @lalunamel this has been removed (to be released). It shouldn't really impact you, but just a heads up.